Program.cs

```diff
@@ -7,19 +7,8 @@
 public partial class MyClass
 {
     [JSExport]
-    internal static string Greeting()
+    internal static Task<string> loadData()
     {
-        var text = $"Hello, World! Greetings from {GetHRef()}";
-        Console.WriteLine(text);
-        return text;
+        return TestLoader.loadTests();
```

Comment:
Perhaps the implementation could be inside here, or perhaps the `[JSExport]` should be in the `TestLoader` class.